cmake/QskBuildFunctions.cmake

```diff
@@ -118,4 +124,9 @@ function(qsk_add_shaders target)
     qt6_add_shaders( ${target} "qskshaders" BATCHABLE PRECOMPILE QUIET
         PREFIX "/qskinny/shaders" ${ARGV} OUTPUTS ${outfiles} )
 
+    cmake_parse_arguments(PARSE_ARGV 1 arg "" "OUTPUT_TARGETS" "")
```

Comment:
Please add one liner comment on what happens here and why

Reply:
added a comment; I used this line from qt6_add_shaders because we want to pass on OUTPUT_TARGETS to the caller of qsk_add_shaders